### DISCUSSION TITLE
Right click tray icon to configure

### DISCUSSION THREAD

**Discussion by leland37** [BazzaCuda/CustomMenu#19] [Ideas]
I hate the fact I have to right click on the desktop to configure Custom Menu.  I rarely look at my desktop and it feels unintuitive especially since you took the time to have a tray icon.  You could add configure as well as exit menus with right click on the tray icon.  This simplicity would likely make it more accessible to new users.  Anyway that is my request.  Custom menu is quite nice other than this and that is the main reason I am only experimenting with it rather than using it on a daily basis.  I look forward to seeing what it can become.

**Comment by BazzaCuda** [comment]
A fair point. I will add a config menu item to the tray icon.

**Reply by BazzaCuda** [reply]
Done!
https://github.com/BazzaCuda/CustomMenu/releases/tag/v1.0.7
